spec/components/stylesheet/manager_spec.rb

```diff
@@ -650,13 +650,20 @@ def write(x)
         t.save!
 
         user_theme.add_relative_theme!(:child, t)
+        default_theme.add_relative_theme!(:child, t)
       end
 
       default_theme.set_default!
 
       StylesheetCache.destroy_all
 
-      Stylesheet::Manager.precompile_css
+      output = capture_output(:stderr) do
```

Comment:
This test does not run by default so you have to trust me that it passes in the local env.